src/Hedgehog/Range.fs

```diff
@@ -54,20 +58,62 @@ module Range =
     //
 
     /// Construct a range which represents a constant single value.
+    ///
+    /// >>> Range.bounds x <| Range.singleton 5
+    /// (5, 5)
+    ///
+    /// >>> Range.origin <| Range.singleton 5
+    /// 5
```

Comment:
I'm curious how this comes out in an IDE / html documentation?

This is actually a supported syntax for haddock, so it renders nicely: http://hackage.haskell.org/package/hedgehog-0.2.2/docs/Hedgehog-Range.html#v:singleton